lib/output/utils.js

```diff
@@ -45,6 +52,16 @@ function initCtorRegistry(globalObject) {
   return ctorRegistry;
 }
 
+// The result is used as the `globals` property of the `options` parameter for `webidl-conversions`:
+function conversionGlobalsFromCtorRegistry(ctorRegistry) {
```

Comment:
Let's stop storing the intrinsics under `"%TypeError"` and instead store them under `"TypeError"`. Then we can get rid of this function entirely and just pass the `ctorRegistry`.